docs/section22/Rule22-4.md

```diff
@@ -0,0 +1,58 @@
+
+# CHW&CW - Rule 22-4  
+
+**Rule ID:** 22-4  
+**Rule Description:** For Baseline chilled water loop that is not purchased chilled water and does not serve any computer room HVAC systems, chilled-water supply temperature shall be reset using the following schedule: 44F at outdoor dry-bulb temperature of 80F and above, 54F at 60F and below, and ramped linearly between 44F and 54F at temperature between 80F and 60F.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.9 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
```

Comment:
system 11 is the computer room system. does that need to be removed from this list?

Reply:
got it, that makes sense

Reply:
as discussed during thee RDS call- if one system in proposed design uses purchased CHW, then the entire baseline building will use P CHW